docs/sources/flow/getting-started/opentelemetry-to-lgtm-stack.md

```diff
@@ -97,7 +97,7 @@ You will implement the following pipelines to send your data to Loki, Tempo, and
 ```
 Metrics: OTel → batch processor → Mimir or Prometheus remote write
 Logs: OTel → batch processor → Loki exporter
-Traces: OTel → batch processor → OTel exporter
+Traces: OTel → batch processor → Tempo exporter
```

Comment:
Actually, the [component](https://grafana.com/docs/agent/next/flow/reference/components/otelcol.exporter.otlp/) can send data to _any_ valid OTLP endpoint (such as the one Tempo exposes), so it's not a Tempo-specific thing!

Reply:
Ok, thanks. The new user wizard suggested that I should point my OTel Collector at Grafana Agent (instead of directly at Grafana), which was made me (as someone new to Grafana) assume that Tempo did not support OTLP directly. I can now see that it does, so will do that.